raiden-ts/src/channels/epics.ts

```diff
@@ -595,21 +595,23 @@ export const channelMonitoredEpic = (
   );
 
 const makeDeposit$ = (
-  tokenContract: HumanStandardToken,
-  tokenNetworkContract: TokenNetwork,
-  sender: Address,
-  address: Address,
-  partner: Address,
+  [tokenContract, tokenNetworkContract]: [HumanStandardToken, TokenNetwork],
+  [sender, address, partner]: [Address, Address, Address],
```

Comment:
Not actually part of this PR so feel free to ignore it.
I think the parameter `address` is not well named. In fact the first 5 parameter are all different Ethereum addresses. So I basically have no idea what `address` should be here. I have to go back to check the call references of this function. Even there the variable is just called `address`. Happily this function (was `channelOpenEpic`) is just forwarding this parameter and has a description for it:  `Our address`. I think this could become improved. But as I said maybe not here.

Reply:
Well, the first 2 parameters are actually `Contract` instances, not Addresses. I agree with you, the name can be improved, but this name come from the epics dependencies `RaidenEpicDeps`, used like that all across the SDK, so it already became a common usecase, and the root for it is that it makes pair with the `Signer` instances: you'll find a `signer` and an `address` on the `deps` directly, which are the actual/effective signer and respective address, and `main: { signer: Signer; address: Address }`, which belongs to the main account, if used. The first Address parameter here actually is the `address` of the `main` account (or dupe of the `address`, if subkey isn't set), as we need to know which allowance to check.